src/EditorFeatures/CSharpTest/Diagnostics/GenerateType/GenerateTypeTests.cs

```diff
@@ -1299,7 +1299,7 @@ internal class T
 {
     public T(out DateTime d)
     {
-        d = default(DateTime);
+        d = default;
```

Comment:
💭 I'm guessing this changed because the default language version went from 7.0 to 8.0, which made default expressions applicable at this location.

Reply:
Yes. I'll add new tests though to assert both versions.

---
In reply to: [185127717](https://github.com/dotnet/roslyn/pull/26405#discussion_r185127717) [](ancestors = 185127717)